eng/deploy.yaml

```diff
@@ -56,33 +56,14 @@ stages:
     displayName: deployment approval (conditional)
     environment: ${{ parameters.DeploymentEnvironment }}
 
-- stage: predeploy
-  displayName: Pre-Deployment
-  pool: server
-  dependsOn:
-  - build
-  - Validate
-  - approval
-  jobs:
-  - job: notifyStartDeployment
-    displayName: Notify deployment start
-    pool: server
-    steps:
-    - task: InvokeRESTAPI@1
```

Comment:
I'm not sure if this is OK to remove, since it might be used for Maestro deployments too?

Reply:
I think this was at least necessary for RolloutScorer. Let me know if ProdCon isn't planning to use results from RolloutScorer anymore for Arcade Services and I'll make sure it gets removed from RolloutScorer.